### PR TITLE
pcp 4.1.1 (new formula)

### PR DESCRIPTION
**Pull request by parthbeingparth**
- [ Yes ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ Yes ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ Yes ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ Yes ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by fxcoudert — CHANGES_REQUESTED** [review]
According to github, the latest stable version is 4.1.0: https://github.com/performancecopilot/pcp/releases

File: Formula/pcp.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class Pcp < Formula
+  desc "Performance Co-Pilot is a system performance and analysis framework"
+  homepage "https://pcp.io/"
+  url "https://dl.bintray.com/pcp/source/pcp-4.1.1.src.tar.gz"
+  sha256 "430c3ce05db5e475dc0d962ac584a61a91c69b791e66d5cc684a7aada3e21f52"
+  depends_on "pkg-config" => :build
```
[the remark this answers is not in the document]
Comment (Warewolf96) [reply Formula/pcp.rb]: I'll rectify this.

File: Formula/pcp.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  depends_on "nss"
+  depends_on "python"
+  depends_on "qt"
+  depends_on "readline"
+  depends_on "xz"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "CC=clang", "CXX=clang", "--without-manager"
+    system "make", "install"
+
```
[the remark this answers is not in the document]
Comment (Warewolf96) [reply Formula/pcp.rb]: Okay

File: Formula/pcp.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  sha256 "430c3ce05db5e475dc0d962ac584a61a91c69b791e66d5cc684a7aada3e21f52"
+  depends_on "pkg-config" => :build
+  depends_on "nss"
+  depends_on "python"
+  depends_on "qt"
+  depends_on "readline"
+  depends_on "xz"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "CC=clang", "CXX=clang", "--without-manager"
```
[the remark this answers is not in the document]
Comment (Warewolf96) [reply Formula/pcp.rb]: It gives an error msg in the log if we don't use CC, CXX as well as --without-manager. I can paste the error log here if you want.

**Pull request closed without merging**